plugins/chat/app/controllers/chat/api/direct_messages_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
```

Comment:
I wonder if its worth creating all this code to delete it soon?

Reply:
I mean, it's just the `with_service` stuff in the controller, the actual service code won't be removed, so IMO it's fine for now.